metadata/hasura.go

```diff
@@ -176,19 +176,23 @@ func (h *Hasura) GetBucketByID(
 }
 
 func (h *Hasura) InitializeFile(ctx context.Context, fileID string, headers http.Header) *controller.APIError {
-	var query struct {
-		InsertStorageFiles struct {
-			ID graphql.String `graphql:"id"`
-		} `graphql:"insertFile (object: {id: $id})"`
+	var result struct {
+		InsertFiles struct {
+			AffectedRows int `graphql:"affected_rows"`
+		} `graphql:"insertFiles"`
 	}
 
 	variables := map[string]interface{}{
 		"id": uuid(fileID),
 	}
 
 	client := h.client.WithRequestModifier(h.authorizer(headers))
-	err := client.Mutate(ctx, &query, variables)
-	if err != nil {
+	if err := client.Exec(
+		ctx,
+		`mutation InitializeFile($id: uuid) {insertFiles(objects: {id: $id}) {affected_rows}}`,
```

Comment:
@dbarrosop I guess the `[]` was not needed?

Reply:
sorry, I thought I had answered and I forgot to press comment.

You don't need the `[]`, otherwise tests would have blown up :P As a matter of fact, that specific query was built by hasura's explorer (by default they don't add the `[]`, you have to do it "manually" in the editor).